x/sync/sync_test.go

```diff
@@ -695,8 +695,11 @@ func TestFindNextKeyRandom(t *testing.T) {
 		}
 
 		// Sort in ascending order by key prefix.
-		serializedPathLess := func(i, j keyAndID) bool {
-			return i.key.Less(j.key)
+		serializedPathLess := func(i, j keyAndID) int {
```

Comment:
I think we'd probably want to rename this var now that it is a comparison rather than just a `less` check